ajax/libs/color-scheme/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "color-scheme",
+  "filename": "color-scheme.min.js",
+  "version": "1.0.0",
+  "description": "Generate pleasant color schemes",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/c0bra/color-scheme-js.git"
+  },
+  "homepage": "http://c0bra.github.io/color-scheme-js/",
+  "keywords": [
+    "color",
+    "scheme",
+    "generator",
+    "random",
+    "picker",
+    "web"
+  ],
+  "npmName": "color-scheme",
+  "npmFileMap": [
+    {
+      "basePath": "lib",
+      "files": [
+        "color-scheme*"
+      ]
+    }
+  ],
+  "author": "Brian Hann",
+  "license": "MIT"
```

Comment:
Just for the sake of uniformity, we keep the auto-update section at the end of the file. Shift `author` and `license` after `description` please.

PS: Apologies for missing this out earlier.